weaver/wps_restapi/swagger_definitions.py

```diff
@@ -2788,17 +2801,19 @@ class PostProcessJobsEndpoint(ProcessPath):
 
 class GetJobsQueries(ExtendedMappingSchema):
     detail = ExtendedSchemaNode(Boolean(), description="Provide job details instead of IDs.",
-                                default=False, example=True, missing=drop)
+                                default=False, example=True, missing=False)
     groups = ExtendedSchemaNode(String(),
                                 description="Comma-separated list of grouping fields with which to list jobs.",
-                                default=False, example="process,service", missing=drop)
-    page = ExtendedSchemaNode(Integer(), missing=drop, default=0, validator=Range(min=0))
-    limit = ExtendedSchemaNode(Integer(), missing=drop, default=10)
+                                default=False, example="process,service", missing=False)
+    page = ExtendedSchemaNode(Integer(), missing=0, default=0, validator=Range(min=0))
+    limit = ExtendedSchemaNode(Integer(), missing=10, default=10, validator=Range(min=0, max=10000))
+    datetime_interval = DATETIME_INTERVAL(missing=None, default=None)
```

Comment:
Should be only `datetime` as per https://github.com/opengeospatial/ogcapi-common/blob/Working/collections/requirements/collections/REQ_rc-time-collections-response.adoc